client/src/pages/Feed.js

```diff
@@ -535,8 +541,9 @@ const Feed = (props) => {
   };
 
   const loadPosts = useCallback(async () => {
+    dispatchAction(SET_VALUE, "applyFilters", false);
```

Comment:
remove this line to fix pagination. right now it doesn't load the next page

Reply:
removing that breaks the search tabs somehow. I foud another solution by setting `applyFilters` to `true` on `loadNextPage()` which will be again turned back to false on `loadPosts()` and nothing breaks.

Reply:
Yes, this one would work too.